src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -241,6 +241,15 @@
 WINDOWS_CDB_STACK_OVERFLOW_REGEX = re.compile(
     r'.*ExceptionCode: .*\(Stack overflow\).*')
 
+# Golang specific regular expressions.
+GOLANG_PANIC_REGEX = re.compile(
+    r'^(panic: runtime error|fatal error): (index out of range|'
+    r'integer divide by zero|invalid memory address|'
+    r'makeslice: len out of range|slice bounds out of range|stack overflow|'
+    r').*')
+GOLANG_STACK_FRAME_FUNCTION_REGEX = re.compile(
+    r'^([0-9a-zA-Z\.\-\_\\\/\(\)\*]+)\([0-9a-zA-Z\s,\_]*\)$')
```

Comment:
This regex can impact other non-go things and can add weird things to their crash state. we need to think if this regex can be made more specific in style and other thing we can do to limit damage is

```
    if golang_panic_match:
      state.crash_type = golang_panic_match.group(2)
      is_go_crash = True
      continue

and then in lower one, do like
if is_go_crash and add_frame_on_match

Reply:
Good call! The panic signature is pretty generic and might be similar to non-Go, but I'll check the presence of the go sources in the stacktrace -- should be safe.

Reply:
I think I can improve the regex itself a bit more -- the arguments are either empty, or hex numbers, or `...`. Updating now.